test/tests.py

```diff
@@ -1,29 +1,161 @@
 import unittest
 import os
 import sys
+import csv
+import pickle
+import json
 from os import path
-sys.path.insert(0, '../lib')
-sys.path.insert(0, '../in_out')
-sys.path.append(path.dirname(path.dirname(path.abspath(__file__))))
-from lib.analysis import network
-from lib import config, nickTracker
+sys.path.insert(0,'../lib')
+sys.path.insert(0,'../in_out')
+sys.path.append( path.dirname( path.dirname( path.abspath(__file__) ) ) )
+from lib.analysis import network, user, channel
+from lib import config, nickTracker, util
 from in_out import reader
 import networkx as nx
-test_dir = os.path.dirname(__file__)
 
-log_directory = os.path.join(test_dir, 'data/input/')
-channel_name = config.CHANNEL_NAME
-starting_date = config.STARTING_DATE
-ending_date = config.ENDING_DATE
-output_directory = os.path.join(test_dir, 'data/output/')
+dir = os.path.dirname(__file__)
 
+log_directory = os.path.join(dir, 'data/input/')
+output_directory = os.path.join(dir, 'data/output/')
+channel_name = config.CHANNEL_NAME
+starting_date = "2013-1-1"
+ending_date = "2013-8-31"
 log_data = reader.linux_input(log_directory, channel_name, starting_date, ending_date)
 nicks, nick_same_list = nickTracker.nick_tracker(log_data)
 
+def compare_graph_outputs (generated_output, stored_output_file_name):
+
+    expected_output = nx.read_gpickle(output_directory+stored_output_file_name)
+
+    if(nx.is_isomorphic(generated_output, expected_output)):
+        return True
+
+    return False
+
+def csv_to_list (csv_path,gen_list, data_in_tuples = True):
+
```

Comment:
don't add extra enters. It makes very difficult to read the code.
and use 2 enters between function definitions. Aren't you using a linter?